plugin/builtin/namespacetransformer/NamespaceTransformer_test.go

```diff
@@ -158,7 +158,7 @@ webhooks:
 - clientConfig:
     service:
       name: svc1
-      namespace: test
+      namespace: system
```

Comment:
Can you add the name  reference transformer in this test? It is to make sure the output is the same as previous after both namespace and name reference transformer.

Reply:
@Liujingfang1 This is what I did in the pkg/target/namespace_test.go: https://github.com/kubernetes-sigs/kustomize/pull/1379/commits/2e46bc8d0574e8d141a328eeb782da2fb7b233e1. I'm updating the commit to add clarification regarding the change.

Reply:
@Liujingfang1. Done.  Added a comment in that test suite. We have a new test suite involving nameprefix, namespace and namereference in pkg/target